lib/install_go.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Borrowing this script from the Ruby Buildpack.  
```

Comment:
Should this say the "Node.js" buildpack instead of "Ruby"?

```suggestion
# Borrowing this script from the Node.js Buildpack.  
```

Reply:
Fixed on next commit.